tests/Storage.test.ts

```diff
@@ -360,46 +382,180 @@ function getBlockData (ledger_storage: LedgerStorage, height: any, callback?: an
+                '0x3a245017fee266f2aeacaa0ca11171b5825d34814bf1e33fae76cca50751e5c' +
+                'fb010896f009971a8748a1d3720e33404f5a999ae224b54f5d5c1ffa345c046f7');
+
+            ledger_storage.getTxInputs(1, 0, (err4: Error | null, rows4: any[]) =>
+            {
+                assert.ok(!err4);
+                assert.equal(rows4.length, 1);
+                assert.equal(rows4[0].previous,
+                    '0x5d7f6a7a30f7ff591c8649f61eb8a35d034824ed5cd252c2c6f10cdbd223671' +
+                    '3dc369ef2a44b62ba113814a9d819a276ff61582874c9aee9c98efa2aa1f10d73');
+
+                ledger_storage.getTxOutputs(0, 1, (err5: Error | null, rows5: any[]) =>
+                {
+                    assert.ok(!err5);
+                    assert.equal(rows5.length, 8);
+                    assert.equal(rows5[0].utxo_key,
+                        '0xef81352c7436a19d376acf1eb8f832a28c6229885aaa4e3bd8c11d5d072e160' +
+                        '798a4ff3a7565b66ca2d0ff755f8cc0f1f97e049ca23b615c85f77fb97d7919b4');
+                    assert.equal(rows5[0].tx_hash,
+                      '0x5d7f6a7a30f7ff591c8649f61eb8a35d034824ed5cd252c2c6f10cdbd223671' +
+                      '3dc369ef2a44b62ba113814a9d819a276ff61582874c9aee9c98efa2aa1f10d73');
+                    assert.equal(rows5[0].address, 'GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ');
+                    assert.equal(rows5[0].used, 1);
+
+                    callback();
+                });
+            });
+        });
     });
+}
+
+/**
+ * Puts all transactions data
+ */
+function putAllTransactionData (ledger_storage: LedgerStorage,
+    callback: (err: Error | null) => void)
+{
+    var idx = 0;
+    var doPut = () =>
+    {
+        if (idx >= sample_data.length)
+        {
+            callback(null);
+            return;
+        }
+
+        ledger_storage.putTransactions(sample_data[idx], (err: Error | null) =>
+        {
+            if (!err)
+            {
+                idx++;
+                doPut();
+            }
+            else
+            {
+                callback(err);
+            }
+        });
+    }
+    doPut();
+}
 
-    it('Test enrollment storage and inquiry function.', () =>
+/**
+ * Run Validators API tests
+ */
+function runValidatorsAPITest (ledger_storage: LedgerStorage, callback?: (err: Error | null, rows: any[]) => void)
+{
+  var address: string = 'GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2OKDM2VJ235GN';
+  var res = ledger_storage.getValidatorsAPI(1, null, (err1: Error | null, rows: any[]) =>
+  {
+      assert.ok(!err1, err1?.message);
+      assert.equal(rows[0].address, address);
+      assert.equal(rows[0].enrolled_at, 0);
+      assert.equal(rows[0].distance, 0);
+
+      var res1 = ledger_storage.getValidatorsAPI(1, address, (err2: Error | null, rows: any[]) =>
+      {
+          assert.ok(!err2, err2?.message);
+          assert.equal(rows.length, 1);
+          assert.equal(rows[0].address, address);
```

Comment:
```suggestion
          assert.equal(rows[0].address, address);
          assert.equal(rows[0].stake, '0x210b66053c73e7bd7b27673706f0272617d09b8cda76605e91ab66ad'+
          '1cc3bfc1f3f5fede91fd74bb2d2073de587c6ee495cfb0d981f03a83651b48ce0e576a1a');
```
I'd like you to check on `stake`, too.

Reply:
@MichaelKim20 Thanks you